src/Umbraco.Web.UI.Login/src/utils/load-custom-view.function.ts

```diff
@@ -0,0 +1,29 @@
+import { html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+/**
+ * Try to load a custom view.
+ * Supports both HTML and JS files (JS files must have a default export).
```

Comment:
You use HTML extensions today to render an AngularJS controller. We didn't want to remove that type of extension in case you just want to show a simple button and to avoid throwing errors when you try and load an older AngularJS view.